smooks/1.5.1.wso2v3/pom.xml

```diff
@@ -0,0 +1,208 @@
+                    <artifactId>jta</artifactId>
+                </exclusion>
+                <exclusion>
+                    <groupId>org.codehaus.groovy</groupId>
+                    <artifactId>groovy-all</artifactId>
+                </exclusion>
+                <exclusion>
+                    <groupId>org.apache.ibatis</groupId>
+                    <artifactId>ibatis-sqlmap</artifactId>
+                </exclusion>
+                <exclusion>
+                    <groupId>org.codehaus.jackson</groupId>
+                    <artifactId>jackson-core-lgpl</artifactId>
+                </exclusion>
+                <exclusion>
+                    <groupId>javax.persistence</groupId>
+                    <artifactId>persistence-api</artifactId>
+                </exclusion>
+                <exclusion>
+                    <groupId>net.sf.opencsv</groupId>
+                    <artifactId>opencsv</artifactId>
+                </exclusion>
+                <exclusion>
+                    <groupId>ognl</groupId>
+                    <artifactId>ognl</artifactId>
+                </exclusion>
+                <exclusion>
+                    <groupId>javassist</groupId>
+                    <artifactId>javassist</artifactId>
+                </exclusion>
+            </exclusions>
+        </dependency>
+    </dependencies>
+    <distributionManagement>	
+        <repository>
+            <id>wso2.releases</id>
+            <name>WSO2 internal Repository</name>
+            <url>http://maven.wso2.org/nexus/content/repositories/releases/</url>
+        </repository>
+    </distributionManagement>
+    <build>
+        <plugins>
+            <plugin>
+                <groupId>org.apache.felix</groupId>
+                <artifactId>maven-bundle-plugin</artifactId>
+                <extensions>true</extensions>
+                <configuration>
+                    <instructions>
+                        <Bundle-SymbolicName>${project.artifactId}</Bundle-SymbolicName>
+                        <Bundle-Name>${project.artifactId}</Bundle-Name>
+                        <Private-Package>
+                            org.thoughtworks.xstream.*;version="${org.thoughtworks.xstream.version}",
+                            org.mvel.*;version="${org.mvel.version}",
+                            freemarker.*;version="${freemarker.version}",
+                            org.yaml.*;version="${org.yaml.version}",
+                            org.mvel2.*;version="${org.mvel2.version}"
+                        </Private-Package>
+                        <Export-Package>
+                            META-INF.xsd.smooks,
+                            META-INF.xsd.smooks.*;-split-package:=merge-first,
+                            org.milyn.*;version="${org.milyn.osgi.version}"
+                        </Export-Package>
+                        <Import-Package>
+                            au.com.bytecode.opencsv;resolution:=optional,

```

Comment:
Most of the import packages does not have version (range). The best practice is to use proper import versions.